docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,111 @@
+
+## Prerequisites
+
+Before you can launch a peer node in a production network, you need to configure it. Complete the following steps:
+
+1. Create the recommended folder structure for the peer certificates that is described in the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic to store the generated certificates and MSPs:
+    ```
+    ├── organizations
+      └── peerOrganizations
+          └── org1.example.com
+              ├── msp
+              └── peers
+                  └── peer0.org1.example.com
+                      ├── msp
+                      └── tls
+    ```
+2. While **cryptogen** is a convenient utility that can be used to generate certificates for a test network, it should **never** be used on a production network. The core requirement for certificates for Fabric nodes is that they are Elliptic Curve (EC) certificates. You can use any tool you prefer to issue these certificates (for example, OpenSSL). However, the Fabric CA streamlines the process because it generates the Membership Service Providers (MSPs) for you. Regardless of which certificate provider you choose, you should have already used it to generate the peer enrollment certificate, TLS certificate, private keys, and the MSPs that Fabric must consume. Refer to the [CA deployment](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/cadeploy.html) and [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topics for instructions on how to generate these certificates. You need to generate the following sets of certificates:
+    - **Peer TLS certificates**
+    - **Peer organization admin certificates (organization MSP)**
+    - **Peer enrollment certificates (peer local MSP)**  
+3. You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, must users are ultimately likely to script this process so it can be repeatable as needed. A list of the certificates and their locations is provided here for your convenience.
+      - **Peer TLS certificate and private key**:
+        - Copy the TLS CA TLS certificate `tls-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/tls-cert.pem`. The path and name of the certificate corresponds to `tls.rootcert.file` parameter in `core.yaml`.
```

Comment:
peer.tls.rootcert.file default value is "tls/ca.crt". Are you recommending it be overridden?
It is the "TLS CA root certificate", not "TLS CA TLS certificate".
Therefore, copy tls ca.crt file to this location (or whatever file name you want to use).
Note that organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls is incorrect here... that location is only used in "test network". The deployment guide is not for test network.

Reply:
@pamandrejko Could you weigh in here? Sounds like you transposed this section (which refers to tls.rootcert.file) with the one below it (which talks about the tls root cert).

Reply:
@denyeart  I reviewed this and what I have here is intentional, so correct me if I am wrong. I was using the same naming convention we used in the CA deployment guide - see the diagram at the end of this [topic](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/cadeploy.html#enroll-bootstrap-user-with-tls-ca) which exactly matches the default names of the certs that are generated. 

This is my experience/understanding:

1. `ca-cert.pem`- When you initially start up  a CA server, the file `ca-cert.pem` is generated. This is the self-signed root CA cert. And if this is a TLS CA, it would be the "TLS CA root cert". This file corresponds to the `tls.clientRootCAs.files` parameter.

2. `tls-cert.pem` - When you enable TLS communications on the CA, the file `tls-cert.pem` is generated. (Also why I called it the TLS CA TLS cert which is a horrible name I know).  This file corresponds to the `tls.rootcert.file` parameter. So it does seem like we have a mismatch with your default value - tls/ca.crt. 

I replicated these steps this morning on a new CA server, just to double-check the file naming to make sure I did not transpose anything.  Feel free to correct me.

Reply:
I checked with Saad on this one. I stand corrected:

1) The file that needs to go in `tls.rootcert.file` is the cert that contains the public key that is associated with the signing (private) key, which in a default CA scenario is going to be ca-cert.pem from my example above.

2) `tls.clientRootCAs.files` is only required when mutual TLS is configured. So for tls.clientRootCAs.files it should be the certificate that will be used to verify the client certificate, which file goes there all depends on who is issuing the client’s TLS certificate. Since we are not using mutual TLS in this tutorial, it does not need to be included here.